common/middleware/questionGroups/postHandlers.js

```diff
@@ -39,78 +36,6 @@ const assembleDates = async (req, res, next) => {
   return next()
 }
 
-const constructValidationRule = (questionId, validationType, validationSettings) => {
-  switch (validationType) {
-    case 'mandatory':
-      return body(questionId)
-        .isLength({ min: 1 })
-        .withMessage({ error: validationSettings.errorMessage, errorSummary: validationSettings.errorSummary })
-    default:
-      return ''
-  }
-}
-
-const questionGroupValidationRules = async (req, res, next) => {
-  const {
-    body: reqBody,
-    params: { tableName },
-  } = req
-
-  const { questionGroup } = res.locals
-  let currentQuestions = questionGroup.contents
-
-  if (tableName) {
-    currentQuestions = currentQuestions.find(element => element.tableCode === tableName).contents
-  }
-
-  const validatorsToSend = []
-
-  currentQuestions.forEach(question => {
```

Comment:
QQ - Just check my understanding here - wondering if removing this would affect validation being triggered for conditional questions? Or are we handling this using custom validators in `fields.js`?

Reply:
I've checked and all the conditional validation in UPW still seems to work fine. It's a whole different mechanism we're using for validation now.

Reply:
👍